lib/util.js

```diff
@@ -279,11 +279,17 @@ const util = {
     util.run('ninja', ['-C', config.outputDir, config.buildTarget, '-k', num_compile_failure], options)
   },
 
-  lint: (options = {}) => {
+  lint: (options) => {
     let cmd_options = config.defaultOptions
+    let base_branch = 'origin/master'
     cmd_options.cwd = config.projects['brave-core'].dir
+    if (options) {
+        base_branch = options;
+    }
     util.run('python', [path.join(config.rootDir, 'scripts', 'lint.py'),
-        '--project_root=' + config.srcDir], cmd_options)
+	'--project_root=' + config.srcDir,
+        '--base_branch=' + base_branch,
+        '--current_branch=' + config.projects['brave-core'].ref], cmd_options)
```

Comment:
why is this even necessary? Also using `.ref` here isn't correct because we don't always set `.ref`

Reply:
@bridiver - `git diff --name-only` needs two branches to get the list of changed files. I can modify the code to fetch the `current_branch` as a param to the lint script. I just used the param from package.json, since its already there.